rel/files/riak_cs.schema

```diff
@@ -671,6 +671,41 @@
   {default, "{{platform_log_dir}}"}
 ]}.
 
+%% == Supercluster aka Multi-bug Support ==
+
+%% @doc IP and port for supercluster members.  Specify one member for
+%% each line, in which the last token of key is member identifier and
+%% value is a PB address of Riak node to connect.
+{mapping, "supercluster.member.$member_id", "riak_cs_multibag.bags", [
+  {datatype, ip},
+  {include_default, "bag-A"},
```

Comment:
The best way is to match the string here with one in the doc (in the future.) This is *not user-facing* because of `hidden`, how about leaving as it is and defer change ?